logic_mod/Blocks/ExtSpeedometerBlock.cs

```diff
@@ -95,6 +95,17 @@ protected override void Awake()
             speedSlider.DisplayName = ">" + speedSlider.DisplayName;
         }
 
+        public Dictionary<string, object> GetV()
+        {
+            var myV = Rigidbody.velocity;
+            return new Dictionary<string, object>
+            {
+                { "x", myV.x },
+                { "y", myV.y },
+                { "z", myV.z }
+            };

```

Comment:
The helper function converting Vector3->Dictionary, made for anglometer, also should be used here